appinterface/projection/crossfire/httpclient_test.go

```diff
@@ -23,7 +23,7 @@ var _ = Describe("Crossfire Chain Stats", func() {
 			1,
 			"foo",
 			"14",
-			"https://raw.githubusercontent.com/foreseaz/random/master/participants.json",
+			"https://raw.githubusercontent.com/foreseaz/random/master/",
```

Comment:
Maybe it's better to keep the original full URL and it's better to replace them as a whole.

Reply:
https://github.com/crypto-com/chain-indexing/pull/264#discussion_r553123893